content/en/dashboards/guide/apm-stats-graph.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Configuring an APM stats graph
+kind: guide
+disable_toc: false
+aliases:
+- /dashboards/querying/#configuring-an-apm-stats-graph
+further_reading:
+- link: "/dashboards/querying/"
+  tag: "Documentation"
+  text: "Querying graphs"
+---
+
+To configure your graph using APM stats data, follow these steps:
+
+1. [Select your visualization](#select-your-visualization) (same as for Metrics)
+2. [Choose your level of detail](#level-of-detail)
+3. [Choose your parameters](#apm-stats-parameters)
+4. [Title the graph](#create-a-title) (same as for Metrics)
```

Comment:
I believe these steps are repeated in https://github.com/DataDog/documentation/pull/18005/files#diff-62c7e5fe878506604440adcc2c67d8ecec46dade159d834db0888bc3d4d14791R35-R41? Should they be permanent links with anchors instead, or do you want to move the content out of the Querying doc page into this guide?

Reply:
Ooh, thanks for catching this, copy pasta error. I'll link out to the Querying page.